WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderSearchSuggestionAdapter.java

```diff
@@ -1,53 +1,58 @@
 package org.wordpress.android.ui.reader.adapters;
 
+import android.app.AlertDialog;
 import android.content.Context;
+import android.content.DialogInterface;
 import android.database.Cursor;
+import android.database.MatrixCursor;
 import android.support.v4.widget.CursorAdapter;
-import android.text.TextUtils;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.widget.ImageView;
 import android.widget.TextView;
 
 import org.wordpress.android.R;
 import org.wordpress.android.datasets.ReaderSearchTable;
 
 public class ReaderSearchSuggestionAdapter extends CursorAdapter {
     private static final int MAX_SUGGESTIONS = 5;
+    private static final int CLEAR_ALL_ROW_ID = -1;
+
     private String mCurrentFilter;
```

Comment:
> mCurrentFilter isn't being read anywhere. Can it be removed?

Glad you saw that, because it alerted me to another problem. When `setFilter` is called, it should skip recreating the cursor if it's the same as the current filter.

This check exists [in the master branch](https://github.com/wordpress-mobile/WordPress-Android/blob/feature/reader-search-master/WordPress/src/main/java/org/wordpress/android/ui/reader/adapters/ReaderSearchSuggestionAdapter.java#L29) but I must've deleted it in this branch by mistake.

Fixed in 87c2fa0.